src/Page/Page.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react"
 
-import Tabs, { Tab, tabsBarHeight } from "../Internals/Tabs"
+import Tabs, { Tab, tabsBarHeight, tabsBarMargin } from "../Internals/Tabs"
```

Comment:
This value comes from the theme.

```suggestion
import Tabs, { Tab, tabsBarHeight } from "../Internals/Tabs"
```

Reply:
Related discussion: https://github.com/contiamo/operational-ui/pull/1018#discussion_r292553119